### DISCUSSION TITLE
README getting truncated

### DISCUSSION THREAD

**Discussion by youjinp** [community/community#8222] [Other Feature Feedback, Questions, & Ideas]
The README on my repo ([https://github.com/youjinp/wwdc-list](https://github.com/youjinp/wwdc-list)) is being truncated on the repo's front page.

However, the README is rendered completely on the file's blob page.

This bug has been raised in [https://github.community/t/does-readme-md-have-max-characters/128124](https://github.community/t/does-readme-md-have-max-characters/128124) and [https://github.com/isaacs/github/issues/1610](https://github.com/isaacs/github/issues/1610) but so far haven't gotten any explanation / fix. Thought I'd post it here too.

**Comment by aloisklink** [comment]
This is also affecting at least one big repo with 600+ stars: [https://github.com/gajus/eslint-plugin-jsdoc (commit `747db99`)](https://github.com/gajus/eslint-plugin-jsdoc/tree/747db99b333489aa75765d113c421e66bc77db4a)

Their README.md has grown to 531kB, and now only the first 512kB (512,000 bytes) are being shown.

I'm guessing this truncation is for performance reasons, but it would be nice to at least show a warning that the README.md is truncated.
